src/app.ts

```diff
@@ -26,60 +25,64 @@ const jsonKeyring = readFileSync(KEYRING_PATH).toString();
 const KEYRING = GearKeyring.fromJson(jsonKeyring, KEYRING_PASSPHRASE);
 
 const createVoucher = async (accountUser: HexString) => {
+  const provider = new WsProvider(NODE_ADDRESS);
   const api = await GearApi.create({
-    providerAddress: NODE_ADDRESS,
+    provider
   });
```

Comment:
This is not the best way to connect to the node on every request. You should only have one connection while the backend is running